lib/intro_slider.dart

```diff
@@ -758,6 +765,13 @@ class IntroSliderState extends State<IntroSlider>
     );
   }
 
+  Widget renderBottom() {
+    if (widget.renderCustomBottom != null) {
+      return widget.renderCustomBottom!(renderDotIndicator());
```

Comment:
and here, why we pass `renderDotIndicator()` when we're using custom bottom

Reply:
For example, I would like to custom the `TextButton` to let it have a larger width.

Reply:
When we build our own bottom layout, we may need to reuse the dot indicators. So I put it as a parameter, we can use it or not, it depends.